prow/test/integration/fakegerritserver/fakegerritserver.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// fakegerritserver serves github API for integration tests.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+
+	gerrit "github.com/andygrunwald/go-gerrit"
+	"k8s.io/test-infra/prow/interrupts"
+	"k8s.io/test-infra/prow/logrusutil"
+	"k8s.io/test-infra/prow/pjutil"
+)
+
+type options struct {
+	port int
+}
+
+func (o *options) validate() error {
+	return nil
+}
+
+func flagOptions() *options {
+	o := &options{}
+	flag.IntVar(&o.port, "port", 8888, "Port to listen on.")
+	return o
+}
+
+func main() {
+	logrusutil.ComponentInit()
+
+	o := flagOptions()
+	flag.Parse()
+	if err := o.validate(); err != nil {
+		logrus.WithError(err).Fatal("Invalid arguments.")
+	}
+	defer interrupts.WaitForGracefulShutdown()
+
+	r := mux.NewRouter()
+
+	r.Path("/").Handler(response(defaultHandler()))
+	r.Path("/changes/{change-id}").Handler(response(changesHandler()))
+
+	health := pjutil.NewHealth()
+	health.ServeReady()
+
+	logrus.Info("Start server")
+
+	// setup done, actually start the server
+	server := &http.Server{Addr: ":8888", Handler: r}
```

Comment:
didn't catch this last time, use `o.port` instead

Reply:
Done